BedrockServer.cpp

```diff
@@ -355,12 +352,19 @@ void BedrockServer::sync(SData& args,
             if (nodeState == SQLiteNode::STANDINGDOWN) {
                 continue;
             }
-
             // Now we can pull the next command off the queue and start on it.
             command = syncNodeQueuedCommands.pop();
             SINFO("[performance] Sync thread dequeued command " << command.request.methodLine << ". Sync thread has "
                   << syncNodeQueuedCommands.size() << " queued commands.");
 
+            // Set the function that will be called if this thread's signal handler catches an unrecoverable error,
+            // like a segfault. This version is simpler than the one in `worker`, as no synchronization needs to be
```

Comment:
Is this comment out of date?  The worker and sync thread handler functions seem identical to me.